pyodk/_endpoints/forms.py

```diff
@@ -137,13 +136,8 @@ def update(
                 if not fda.upload(file_path=attach, **fp_ids):
                     raise PyODKError("Form update (attachment upload) failed.")
 
-        # Get a new version - using either a timestamp or the callback.
-        new_version = datetime.now().isoformat()
-        if version_updater is not None:
-            new_version = version_updater(self.get(**fp_ids).version)
+        new_version = datetime.now().isoformat() if definition is None else None
```

Comment:
I think it's better to write as a block for readability and in case other cases are added